src/main/java/com/cloudbees/jenkins/plugins/bitbucket/PullrequestSCMHead.java

```diff
@@ -0,0 +1,37 @@
+package com.cloudbees.jenkins.plugins.bitbucket;
+
+import com.cloudbees.jenkins.plugins.bitbucket.api.BitbucketPullRequest;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import hudson.model.Action;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class PullrequestSCMHead extends SCMHeadWithOwnerAndRepo {
```

Comment:
Before messing around with these classes, it might be wise to take into account [JENKINS-36283](https://issues.jenkins-ci.org/browse/JENKINS-36283), which would probably involve changing the representation of heads and/or revisions. At least it did in the case of the analogous https://github.com/jenkinsci/github-branch-source-plugin/pull/60.